src/pages/404/index.tsx

```diff
@@ -26,6 +29,20 @@ const NotFoundImage = styled.img`
   }
 `
 
+const NotErrorImage = styled.img`
+  width: 602px;
+  height: 273px;
+  margin: 90px auto;
+  display: block;
+  border: 1px dashed #9c9c9c;
+  padding: 6px;
+
+  @media (max-width: 750px) {
+    width: 228px;
+    height: 108px;
+  }
+`
```

Comment:
Styled component should be avoided in further development and will be removed in the future.